2-Regression/translations/README.az.md

```diff
@@ -0,0 +1,39 @@
+
+## Nə öyrənəcəksiniz
+
+[![Reqressiyaya giriş](https://img.youtube.com/vi/5QnJtDad4iQ/0.jpg)](https://youtu.be/5QnJtDad4iQ "Reqressiya giriş videosu - İzləmək üçün klikləyin!")
+> 🎥 Bu dərsə sürətli giriş videosu üçün yuxarıdakı şəkilə klikləyin.
+
+Bu bölmədəki dərslər maşın öyrənməsi konteksti çərçivəsində reqressiyaların növlərini əhatə edir. Reqressiya modelləri dəyişənlər arasındakı _əlaqəni_ təyin etməyə kömək edir. Bu model tipi dəyişənlər arası əlaqəni data nöqtləri vasitəsilə analiz edərək üzə çıxarır və bunun sayəsində uzunluq, temperatur və ya yaş kimi dəyərləri təxmin edə bilir.
+
+Bu dərslər seriyasında, xətti və logistik reqressiyalar arasındakı fərqi, nə zaman hansını seçməli olduğunuzu öyrənəcəksiniz.
+
+[![Yeni başlayanlar üçün MÖ - Maşın Öyrənməsi üçün reqressiya modellərinə giriş](https://img.youtube.com/vi/XA3OaoW86R8/0.jpg)](https://youtu.be/XA3OaoW86R8 "Yeni başlayanlar üçün MÖ - Maşın Öyrənməsi üçün reqressiya modellərinə giriş")
+
+> 🎥 Reqressiya modellərinə giriş üçün yuxarıdakı şəkilin üzərinə klikləyin.
+
+Bu dərslər qrupunda siz data alimləri üçün ümumi mühit olan dəftərçələri idarə etmək üçün Visual Studio Code-u konfiqurasiya etmək daxil olmaqla, maşın öyrənmə tapşırıqlarına başlamaq üçün hazırlaşdırılacaqsınız. Siz maşın öyrənməsi üçün olan Scikit-learn kitabxanası haqqında öyrənəcək və bu fəsildə Reqressiya modellərinə diqqət yetirərək ilk modellərinizi quracaqsınız.
+
+> Reqressiya modelləri ilə işləmək haqqında məlumat almanıza kömək olacaq az-kod yanaşmalı alətlər də mövcuddur. Bu tapşırıq üçün olan [Azure MÖ-i](https://docs.microsoft.com/learn/modules/create-regression-model-azure-machine-learning-designer/?WT.mc_id=academic-77952-leestott) sınaqdan keçirin.
+
+### Dərslər
+
+1. [Ticarət alətləri](1-Tools/README.md)
+2. [Datanın idarə olunması](2-Data/README.md)
+3. [Xətti and polinomial reqressiya](3-Linear/README.md)
+4. [Logistik reqressiya](4-Logistic/README.md)
+
+### Təşəkkürlər
+
+"Reqressiya ilə MÖ" [Jen Looper](https://twitter.com/jenlooper) tərəfindən ♥️-lə yazılmışdır.
+
+♥️ Quizlərə töhvə verənlər:: [Muhammad Sakib Khan Inan](https://twitter.com/Sakibinan) və [Ornella Altunyan](https://twitter.com/ornelladotcom)
+
+Balqabaq data massivi [Kaagle-dakı bu proyekt](https://www.kaggle.com/usda/a-year-of-pumpkin-prices) tərəfindən məsləhət görülmüşdür və məlumatlar Amerika Birləşmiş Ştatlarının Kənd Təsərrüfatı Nazirliyi tərəfindən dərc olunmuş [Xüsusi Bitkilər Terminal Bazarlarının Standart Hesabatlarından](https://www.marketnews.usda.gov/mnp/fv-report-config-step1?type=termPrice) götürülmüşdür. Paylanmanı normallaşdırmaq üçün müxtəlifliyə görə rəng əsaslı bəzi nöqtələr əlavə etmişik. Bu məlumatlar publik domendədir.
```

Comment:
publik -> ictimai?

Reply:
We do have "publik" word in our language. For example, "Publik Hüquqi Şəxs"